pages/common/dolt-sql.md

```diff
@@ -0,0 +1,16 @@
+# dolt sql
+
+> Run a SQL query. Multiple SQL statements must be separated by semicolons.
+> More information: <https://docs.dolthub.com/cli-reference/cli#dolt-sql>.
+
+- Run a single query:
+
+`dolt sql -q "INSERT INTO t values (1, 3);"`
+
+- Run multiple SQL statements:
+
+`dolt sql <<SQL ALTER TABLE t ADD c INT; INSERT INTO t VALUES (1, 2, 1); SQL`
```

Comment:
This bash command isn't valid.

Reply:
I found the command in [this example](https://docs.dolthub.com/sql-reference/version-control/dolt-system-tables#dolt_merge_status)

Apologies if it doesn't work, I'll remove it and if someone has a better example is always welcome to add in the page.